src/tools/unicode-table-generator/src/raw_emitter.rs

```diff
@@ -76,15 +76,15 @@ impl RawEmitter {
 
         writeln!(
             &mut self.file,
-            "static BITSET_CANONICAL: [u64; {}] = [{}];",
+            "const BITSET_CANONICAL: [u64; {}] = [{}];",
```

Comment:
Can static variables not be used in const contexts?

Reply:
When I try changing this back to `static` and run `cargo run --manifest-path ./src/tools/unicode-table-generator/Cargo.toml -- ./library/core/src/unicode/unicode_data.rs; x test library/core` I get the following error:

```
error[E0013]: constant functions cannot refer to statics
   --> library/core/src/unicode/unicode_data.rs:419:14
    |
419 |             &BITSET_CANONICAL,
    |              ^^^^^^^^^^^^^^^^                                                                       |
    = help: consider extracting the value of the `static` to a `const`, and referring to that
```

Reply:
Problem is if llvm doesn't guess that this tables also should be used *as static*, this will involve copying that tables around.

Reply:
Oh, I didn't realize that's a risk when they're only ever passed by reference, especially to an always-inlined function. Worth considering, though!

Reply:
Consider using `const MY_CONST: &'static [u8; N]` which is treated as a static table and can be used in const functions.

Reply:
Thanks for the suggestion! Giving that a try now.

Reply:
It worked :). Resolving.